doc/nrf/config_and_build/configuring_app/core_components.rst

```diff
@@ -0,0 +1,83 @@
+.. _core_components:
+
+Using core firmware components
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| contains several standalone, predefined firmware components that you can reuse during the development of your application.
+These components include applications that can be directly included in your project, samples that might require additional configuration, isolated modules created for specific development scenarios, external modules, and others.
+They can be divided into the following categories:
+
+* Functionality images for a single core - These are applications that provide core functionality but are used as-is during application development possibly with small configuration changes.
+  Two examples of such application are :ref:`bootloader` and :ref:`ug_tfm`.
+* Companion core images - These include applications for companion cores, such as the :ref:`ipc_radio`.
+
+These components are placed in different locations in the code structure because of their origin and purpose.
+For example, some of them are forks of external projects (:ref:`ug_tfm`), while others are specific to the |NCS| (:ref:`bootloader`).
+
+The following table lists the available components that you can reuse.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Component
+     - Purpose
+     - Supported platforms
+     - Location
+     - Integration instructions
+   * - :ref:`ipc_radio`
+     - Companion core image that allows to use the radio peripheral from another core in a multicore device.
+     - | :ref:`nRF54H20 DK <ug_nrf54h20_gs>`
+       | :ref:`nRF5340 DK <ug_nrf5340>`
+       | :ref:`Thingy:53 <ug_thingy53>`
+     - :path:`applications/ipc_radio`
+     - :ref:`Configuration <ipc_radio>`
+   * - :ref:`MCUboot <mcuboot:mcuboot_wrapper>`
+     - External secure bootloader for 32-bit microcontrollers.
+     - :ref:`All supported boards <app_boards_names>`
+     - :file:`ncs/bootloader/mcuboot` (external module) and :file:`ncs/nrf/modules/mcuboot` (integration files)
+     - :ref:`mcuboot:mcuboot_ncs`
+   * - TF-M
+     - Platform security architecture reference implementation aligning with PSA Certified guidelines, enabling chips, Real Time Operating Systems and devices to become PSA Certified.
+     - | :ref:`nRF91x1 Series devices <ug_nrf91>`
+       | :ref:`nRF54L15 PDK <ug_nrf54l>`
+       | :ref:`nRF54H20 DK <ug_nrf54h20_gs>`
+       | :ref:`nRF5340 DK <ug_nrf5340>`
+       | :ref:`Thingy:53 <ug_thingy53>`
+     - :file:`ncs/modules/tee/tf-m`
+     - :ref:`ug_tfm`
+   * - :ref:`bootloader`
+     - Bootloader from Nordic Semiconductor, tailored for the :ref:`immutable bootloader architecture <immutable_bootloader>` of a secure boot chain.
```

Comment:
```suggestion
     - Nordic Secure Immutable Bootloader, tailored for the :ref:`two stage<immutable_bootloader>` bootloader
```

Reply:
The name is already used in the ref title one line above, so I did not use it here.